JeMPI_Apps/JeMPI_LibAPI/src/main/java/org/jembi/jempi/libapi/BackEnd.java

```diff
@@ -455,6 +456,7 @@ private Behavior<Event> postUploadCsvFileHandler(final PostUploadCsvFileRequest
             appendUploadConfigToFile(request.uploadConfig, file);
          }
          Files.copy(file.toPath(), Paths.get("/app/csv/" + file.getName()));
```

Comment:
@MatthewErispe Nothing wrong here but just wondering if we shouldn't combine the 3 file operations into a single move operation? ie instead of copy + move + delete, just a move:
`Files.move(file.toPath(), Paths.get("/app/csv/" + file.getName().replace(".tmp", "")));`

Reply:
@martino-jembi I will try this out and get back to you

Reply:
@martino-jembi please review my change requested